api/scpca_portal/migrations/0049_auto_20240731_1820.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 3.2.25 on 2024-07-31 18:20
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("scpca_portal", "0048_library_has_cite_seq_data"),
+    ]
+
+    operations = [
+        migrations.RenameField(
+            model_name="sample",
+            old_name="age_at_diagnosis",
+            new_name="age",
+        ),
+        migrations.AddField(
+            model_name="sample",
+            name="age_timing",
+            field=models.TextField(
+                choices=[
+                    ("collection", "Collection"),
+                    ("diagnosis", "Diagnosis"),
+                    ("unknown", "Unknown"),
+                ],
+                default="unknown",
+            ),
```

Comment:
looks like we need to remove this migration and run `makemigrations` again

Reply:
Sorry about that. Will take care of this now.